### PR TITLE
SWC-6115: special renderer for the currentVersion if is view

### PR DESCRIPTION
**Pull request by jay-hodgson**
Screen Shot 2022-05-24 at 12 51 27 PM

### CODE REVIEW

File: src/lib/containers/CardContainer.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ export const CardContainer = (props: CardContainerProps) => {
       case FUNDER:
         return <Funder {...props} />
       case GENERIC_CARD:
-        return <GenericCard {...props} />
+        return <GenericCard {...props} queryContext={queryContext} />
```
Comment (jay-hodgson) [inline src/lib/containers/CardContainer.tsx:58]: The GenericCard did not have the queryContext (so the table entity check was failing).  Fixed by explicitly passing it down.

File: src/lib/containers/GenericCard.tsx [hunk 2]
```diff
@@ -145,6 +145,37 @@ export const getValueOrMultiValue = ({
   return { str: value, columnModelType: selectedColumnOrUndefined?.columnType }
 }
 
+export const getColumnIndex = (
```
Comment (jay-hodgson) [inline src/lib/containers/GenericCard.tsx:148]: Refactored, but ended up not reusing.

File: src/lib/containers/GenericCard.tsx [hunk 3]
```diff
@@ -145,6 +145,37 @@ export const getValueOrMultiValue = ({
...
+  columnModels?: ColumnModel[],
+): number | undefined => {
+  return (
+    selectColumns?.findIndex(el => el.name === columnName) ||
+    columnModels?.findIndex(el => el.name === columnName)
+  )
+}
+
+// SWC-6115: special rendering of the version column (for Views)
+export const VersionLabel: React.FC<{
```
Comment (jay-hodgson) [inline src/lib/containers/GenericCard.tsx:160]: New FC to render the Version, and link to the Synapse entity page.

File: src/lib/containers/GenericCard.tsx [hunk 4]
```diff
@@ -529,23 +561,33 @@ export default class GenericCard extends React.Component<
     )
     const values: string[][] = []
     const { secondaryLabels = [] } = genericCardSchemaDefined
+    const isView = table && !isTableEntity(table)
```
Comment (jay-hodgson) [inline src/lib/containers/GenericCard.tsx:564]: Rendering an entity under the Table interface, but it's not a TableEntity.  It must be some type of Entity View.

**Review by yjyoo773 — APPROVED** [review]
LGTM

**Pull request merged**